urllib3/connectionpool.py

```diff
@@ -381,8 +382,22 @@ def _make_request(self, conn, method, url, timeout=_Default,
         log.debug("\"%s %s %s\" %s %s" % (method, url, http_version,
                                           httplib_response.status,
                                           httplib_response.length))
+
+        header_parsing_errors = extract_parsing_errors(httplib_response.msg)
```

Comment:
For me exceptions are something even more special than `logging.error`. So if we will not abort we should not be using them.